addOns/authhelper/src/main/javahelp/org/zaproxy/addon/authhelper/resources/help/contents/auth-req-id.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+Authentication Request Identification
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Authentication Request Identification</H1>
+
+This <a href="authhelper.html">add-on</a> includes a passive scan rule which attempts to identify authentication requests.<br>
+It identifies authentication requests by the presence of commonly used username and password field names.
+It also uses commonly used URL segments to identify more likely authentication requests, 
+and uses commonly used registration URL segments to ignore registration requests.<br><br>
+The rule will not attempt to identify very unusual authentication requests - automation is one of the end goals so false negatives
+(missing unusual authentication requests) is more desirable than false positives (incorrectly identifying an authentication request). 
+<p>
+If this rule identifies an authentication request that is part of the context you have defined then an informational alert will be raised and the 
+authentication method should be correctly set in the context you have created.<br>
+If an authentication request is not identified then check to make sure it is made to a site which is included in the context.<br>
+If it is not then add the site to the context and authenticate again via your browser.
+<p>
+The 'other' field is used to report a set of key-value pairs which can be easily parsed. The current keys supported are:
+<ul>
+<li>userParam
+<li>userValue
+<li>passwordParam
+<li>csrfToken
+</ul>
+There can potentially be multiple csrfTokens.
+<p>
+The rule will currently identify:
+<ul>
+<li>Form-based authentication requests
+<li>JSON-based authentication requests
```

Comment:
Should all these list items have closing tags?

Reply:
It's not necessary.